### ISSUE TITLE
Vacations + Remote Work

### ISSUE THREAD

**Issue by vinitra** [EPIC-guide/epic-guide.github.io#46]
https://epic-guide.github.io/middle/vacations

I think it would be nice to mention here:
1) talk to your prof before putting vacation in the system to check the dates are ok
2) your advisor technically can assign your leftover days from the prev year if you haven't taken them by April, so it's in your interest to take them or make a plan for them
3) some advisors request you to put in regular remote working days into the system

**Comment by svaishnavi411** [comment]
A small comment: I'd not expect to find "Sickness and accidents" or "Parenthood" under "Vacations" - Perhaps change it to something like "Leaves of absence" in line with the EPFL guide?

**Comment by SolalPirelli** [comment]
I strongly suggest mentioning as little formal stuff as possible beyond "talk to your advisor", because often informal agreements are much easier for everyone than having to involve HR. The usual "if you ask, you have to listen to the answer".

**Comment by SolalPirelli** [comment]
@svaishnavi411 I agree the page could have a better name but AFAIK "leave of absence" is a specific term not for vacations but for unpaid leave like going on an internship. Perhaps just "Absences" ?

**Comment by svaishnavi411** [comment]
> @svaishnavi411 I agree the page could have a better name but AFAIK "leave of absence" is a specific term not for vacations but for unpaid leave like going on an internship. Perhaps just "Absences" ?

Agreed, "Absences" works.
